File: selection_Sort.py
# selection sort by python
array=[5,7,4,3,8,5,9,43,46,75,54,756,54]
def Selection_Sort(arr):
    for i in range(len(arr)):
        dd = i
        for j in range(i+1,len(arr),1):
            if arr[j] < arr[i]:
                dd = j
        if dd != i:
            T = arr[i]
            arr[i]= arr[dd]
            arr[dd]= T    
    print(arr)            
Selection_Sort(array)    

File: selection_sort.js
// selection sort by js
var list =[3,7,4,6,4,9,8,23,11,33,21,13]
function selection_Sort(list){
    for(var i = 0; i< list.length;i++){
        var index = i
        for (var j= i+1; j< list.length;j++){
            if(list[j]<list[i]){
                index = j
            }
        }
        if(index != i){
            T = list[i]
            list[i]= list[index]
            list[index]=T
        }
    }
    return list
}
console.log(selection_Sort(list))